src/badguy/dispenser.cpp

```diff
@@ -313,8 +314,15 @@ Dispenser::launch_badguy()
       switch (m_type)
       {
         case DispenserType::DROPPER:
-          spawnpoint = get_anchor_pos (m_col.m_bbox, ANCHOR_BOTTOM);
-          spawnpoint.x -= 0.5f * object_bbox.get_width();
+          if (m_flip == NO_FLIP) {
+            spawnpoint = get_anchor_pos (m_col.m_bbox, ANCHOR_BOTTOM);
+            spawnpoint.x -= 0.5f * object_bbox.get_width();
+          }
+          else {
+            spawnpoint = get_anchor_pos (m_col.m_bbox, ANCHOR_TOP);
+            spawnpoint.y -= m_col.m_bbox.get_height();
+            spawnpoint.x -= 0.5f * object_bbox.get_width();
+          }
```

Comment:
General comment, in SuperTux the formatting should be:
```cpp
if (...)
{
  ...
}
else
{
  ...
}
```

Reply:
I wasn't sure about that. I saw both used in ST, also Hume told me `if`s use this format, but he hasn't touched the code for years. I'll change that when I have time.

Reply:
This style of format is all over `dispenser.cpp`. Fixing that now.